test/test_h_transport_problem.py

```diff
@@ -337,6 +337,35 @@ def test_initialise_exports_multiple_exports_same_species():
     assert Ds[0].x.array[0] == Ds[1].x.array[0]
 
 
+def test_export_resets_quantities():
+    """Test that the export.data and export.t are correctly reset every time a simulation is initiated."""
+    my_mat = F.Material(D_0=1, E_D=0)
+    H = F.Species("H")
+    surf = F.SurfaceSubdomain1D(id=1, x=4)
+
+    my_export = F.SurfaceFlux(
+        field=H,
+        surface=surf,
+    )
```

Comment:
Can you parametrise this test to also check for other derived quantities (eg. TotalVolume)